File: specification/compute/resource-manager/Microsoft.Compute/Compute/stable/2025-01-02/examples-js/diskEncryptionSetExamples/DiskEncryptionSet_Get.js
const { ComputeManagementClient } = require("@azure/arm-compute");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to gets information about a disk encryption set.
 *
 * @summary gets information about a disk encryption set.
 * x-ms-original-file: 2025-01-02/diskEncryptionSetExamples/DiskEncryptionSet_Get.json
 */
async function getInformationAboutADiskEncryptionSet() {
  const credential = new DefaultAzureCredential();
  const subscriptionId = "{subscription-id}";
  const client = new ComputeManagementClient(credential, subscriptionId);
  const result = await client.diskEncryptionSets.get("myResourceGroup", "myDiskEncryptionSet");
  console.log(result);
}

File: specification/compute/resource-manager/Microsoft.Compute/Compute/stable/2026-03-02/examples-java/diskEncryptionSetExamples/DiskEncryptionSet_Get.java

/**
 * Samples for DiskEncryptionSets GetByResourceGroup.
 */
public final class Main {
    /*
     * x-ms-original-file: 2026-03-02/diskEncryptionSetExamples/DiskEncryptionSet_Get.json
     */
    /**
     * Sample code: get information about a disk encryption set.
     * 
     * @param manager Entry point to ComputeManager.
     */
    public static void getInformationAboutADiskEncryptionSet(com.azure.resourcemanager.compute.ComputeManager manager) {
        manager.serviceClient().getDiskEncryptionSets().getByResourceGroupWithResponse("myResourceGroup",
            "myDiskEncryptionSet", com.azure.core.util.Context.NONE);
    }
}
